AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/AWSMutationDatabaseAdapter/AWSMutationDatabaseAdapter+MutationEventSource.swift

```diff
@@ -21,13 +21,15 @@ extension AWSMutationDatabaseAdapter: MutationEventSource {
         let fields = MutationEvent.keys
         let predicate = fields.inProcess == false || fields.inProcess == nil
 
+        // TODO remove this in favor of proper sorting/orderBy API
```

Comment:
Might be worthwhile for us to start linking backlog items into our TODOs, so we don't lose track of them

Reply:
Good call and I actually thought about it. We do have an item for this one, but it's a Pivotal story, which is not public. Do you think we should make these GitHub issues?